src/app/components/Table/index.tsx

```diff
@@ -16,16 +16,28 @@ import { backgroundColorAnimation } from '../../../styles/theme/animations'
 type SkeletonTableRowsProps = {
   rowsNumber: number
   columnsNumber: number
+  verbose: boolean
 }
+
+const StyledTableCell = styled(TableCell, {
+  shouldForwardProp: prop => prop !== 'verbose',
+})<{ verbose?: boolean }>(({ theme, verbose }) => ({
+  ...(!verbose
+    ? {
+        padding: theme.spacing(3),
```

Comment:
Looking at mockups, should't we keep spacing(4) for vertical paddings? 

This value is not longer aligned with theme `MuiTableContainer` marginLeft value.
![Screenshot 2023-06-19 at 09 33 46](https://github.com/oasisprotocol/explorer/assets/891392/94c46b60-b103-4da2-9ea0-22647969948c)

Reply:
Right, good point.